src/main/java/org/wmn4j/notation/Articulation.java

```diff
@@ -12,18 +12,83 @@
 	 */
 	ACCENT,
 
+	/**
+	 * Specifies a breath mark.
+	 */
+	BREATH_MARK,
+
+	/**
+	 * Specifies a caesura marking.
+	 */
+	CAESURA,
+
+	/**
+	 * Specifies a fermata.
+	 */
+	FERMATA,
+
+	/**
+	 * Represents an unspecified articulation marking.
+	 */
+	OTHER_ARTICULATION,
```

Comment:
This could be left out from the Articulation enum because `OTHER_ARTICULATION` is just MusicXML's way to handle custom articulations. For custom articulations and notations a different approach is needed in wmn4j.

Reply:
Good point, I didn't think that one through. I removed `OTHER_ARTICULATION ` now in a separate commit.